app/routes/goal_routes.py

```diff
@@ -1 +1,75 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, abort, make_response
+import requests
+import os
+from .route_utilities import validate_model, create_model
+from ..models.goal import Goal
+from ..models.task import Task
+from ..db import db
+
+
+bp=Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+@bp.get("")
+def get_goals():
+    query=db.select(Goal)
+    goals = db.session.scalars(query.order_by(Goal.id))
+    goals_response=[]
+    for goal in goals:
+        goals_response.append(goal.to_dict())
+    return goals_response
```

Comment:
This would be a good place for list comprehension to make the route more concise.
```suggestion
    return [goal.to_dict for goal in goals]
```